### DISCUSSION TITLE
Embed a existing notebook from content in REPL

### DISCUSSION THREAD

**Discussion by delongZhai** [jupyterlite/jupyterlite#1363] [Q&A]
good morning,

thank you for this amazing project.

I have a question about how to customized the REPL application that I saw that's been embedded to various application.

I want to customize the REPL, so that it has the code content, but not auto-run. Is there any way for this to be applicable?

Thank you again for your answers!

**Comment by jtpio** [comment]
> I want to customize the REPL, so that it has the code content, but not auto-run. Is there any way for this to be applicable?

Not at the moment. But https://github.com/jupyterlite/jupyterlite/pull/525 (or similar) might help with this at some point.

Also you may want to embed a notebook instead of the REPL application, as this would allow setting code content in cells.
